docs/deploy-development-rp.md

```diff
@@ -415,3 +415,51 @@ To run fake metrics socket:
 ```bash
 go run ./hack/monitor
 ```
+## Run the RP and create a Hive cluster
+
+**Steps to perform on Mac**
+
+1. Mount your local MacOS filesystem into the podman machine:
+`podman machine init --now --cpus=4 --memory=4096 -v $HOME:$HOME`
```

Comment:
Quick nitpick : All the cli commands added in the doc should be kept inside bash syntax lexer. It will be easier to copy/paste while setting up. Example 

```bash
podman machine init --now --cpus=4 --memory=4096 -v $HOME:$HOME
```

Single command in one block is better, however commands that could be run consecutively can be clubbed into a single highlighted block.